src/Sentry/SentryOptions.cs

```diff
@@ -679,6 +681,33 @@ public StackTraceMode StackTraceMode
         /// </summary>
         public bool KeepAggregateException { get; set; }
 
+        /// <summary>
+        /// Adds a <see cref="JsonConverter"/> to be used when serializing or deserializing
+        /// objects to JSON with this SDK.  For example, when custom context data might use
+        /// a data type that requires custom serialization logic.
+        /// </summary>
+        /// <param name="converter">The <see cref="JsonConverter"/> to add.</param>
+        /// <remarks>
+        /// This currently modifies a static list, so will affect any instance of the Sentry SDK.
+        /// If that becomes problematic, we will have to refactor all serialization code to be
+        /// able to accept an instance of <see cref="SentryOptions"/>.
+        /// </remarks>
+        public void AddJsonConverter(JsonConverter converter)
+        {
+            // protect against null because user may not have nullability annotations enabled
+            if (converter == null!)
+            {
+                throw new ArgumentNullException(nameof(converter));
+            }
+
+            // only add if we don't have this instance already
+            var converters = JsonExtensions.SerializerOptions.Converters;
+            if (!converters.Contains(converter))
```

Comment:
wondering if this should throw?

Reply:
Nah. This way if the SDK gets initialized more than once, it will just skip over adding the duplicate converter instance.